online-unsafe-recovery.md

```diff
@@ -158,12 +164,36 @@ If an error occurs during the task, the last stage in the output shows `"Unsafe
 
 ### Step 3. Check the consistency of data and index (not required for RawKV)
 
-After the recovery is completed, the data and index might be inconsistent. Use the SQL commands [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md), `ADMIN RECOVER`, and `ADMIN CLEANUP` to check the consistency of the affected tables (you can get IDs from the output of `"Unsafe recovery finished"`) for data consistency and index consistency, and to recover the tables.
-
 > **Note:**
 >
 > Although the data can be read and written, it does not mean that there is no data loss.
 
+After the recovery is completed, the data and index might be inconsistent. Use the SQL command [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md) to check the data and index consistency of the affected tables
+
+```sql
+ADMIN CHECK TABLE table_name;
+```
+
+If there are inconsistent indexes, you can fix the index inconsistency by renaming the old index, creating a new index, and droping the old index.
+
+Rename the old index:
+
+```sql
+ALTER TABLE table_name RENAME INDEX index_name TO index_name_lame_duck;
+```
+
+Create a new index:
+
+```sql
+ALTER TABLE table_name ADD INDEX index_name (column_name);
+```
+
+Drop the old index:
+
+```sql
+ALTER TABLE table_name DROP INDEX index_name_lame_duck;
+```
```

Comment:
```suggestion
1. Rename the old index:

    ```sql
    ALTER TABLE table_name RENAME INDEX index_name TO index_name_lame_duck;
    ```

2. Create a new index:

    ```sql
    ALTER TABLE table_name ADD INDEX index_name (column_name);
    ```

3. Drop the old index:

    ```sql
    ALTER TABLE table_name DROP INDEX index_name_lame_duck;
    ```
```